packages/edit-site/src/components/page-pages/index.js

```diff
@@ -32,12 +33,14 @@ export default function PagePages() {
 		pageSize: PAGE_SIZE_VALUES[ 0 ],
 	} );
 	// Request post statuses to get the proper labels.
-	const [ postStatuses, setPostStatuses ] = useState( EMPTY_ARRAY );
-	useEffect( () => {
-		apiFetch( {
-			path: '/wp/v2/statuses',
-		} ).then( setPostStatuses );
-	}, [] );
+	const { records: statuses } = useEntityRecords( 'root', 'status' );
```

Comment:
I wanted to optimize the status retrieval to:

- Only return `name` and `slug`, not all schema fields.
- Only return the statuses we actually expect to use (`publish`, `draft`), as the page entity only requests pages with those statuses.

Though it doesn't seem the [statuses endpoint](https://developer.wordpress.org/rest-api/reference/post-statuses/#retrieve-a-status) supports that kind of filtering.

Reply:
Sometimes it's better to load the full object for performance, that way it's cached and shared with other usage of the same entity.

Not sure it applies here but yeah, this seems fine to me I think